engine/src/CalciteExpressionParsing.cpp

```diff
@@ -493,7 +519,24 @@ cudf::experimental::aggregation::Kind get_aggregation_operation(std::string oper
 //	}
 
 	throw std::runtime_error(
-		"In get_aggregation_operation function: aggregation type not supported, " + operator_string);
+		"In get_aggregation_operation_for_groupby function: aggregation type not supported, " + operator_string);
+}
+
+cudf::experimental::reduction_op get_aggregation_operation_for_reduce(std::string operator_string) {
```

Comment:
This returns a reduction type (as in SUM, MIN, MAX) not a datatype which is what `get_aggregation_output_type` does. And COUNT is missing because the cudf reduction operator does not support COUNT. I am guessing because count is just `size() - null_count()`